app/assets/javascripts/discourse/models/composer.js.es6

```diff
@@ -1097,6 +1099,18 @@ Composer.reopenClass({
     return Object.keys(_create_serializer);
   },
 
+  serializeToDraft(fieldName, property){
```

Comment:
All javascript extensibility should go via the plugin-api so it can be versioned and so we can support backwards compatibility if need be.

Reply:
The methods I referred here were the ember side Category model methods `serializeOnCreate` and `serializeToTopic` and thought that this method should be here itself. Is it fine if I create an alias of the same method for `plugin-api` which essentially calls this method? This would solve the versioning and backwards compatibility problems.

Reply:
Yes creating the alias is fine, and all plugins should access them that way.